src/docker-osx-dev

```diff
@@ -379,13 +384,63 @@ function init_boot2docker {
     log_info "Starting Boot2Docker VM"
     boot2docker start --vbox-share=disable
   fi
+
+  configure_boot2docker_vars
+}
+
+################################################################################
+# docker-machine manipulation
+################################################################################
+
+function get_json_val () { 
```

Comment:
Agreed. I will change to `get_json_value` and add some docs, but I still have work in the python dependency.